NetCoreRestApi/DataLayer/Repositories/CategoryRepository.cs

```diff
@@ -2,10 +2,20 @@
 using System.Collections.Generic;
 using System.Linq;
 
-namespace BusinessLayer.Manager
+namespace DataLayer.Repositories
 {
-    public class CategoryManager : ICategoryManager
+    public class CategoryRepository : IRepository<CategoryModel>
     {
+        public void Create(CategoryModel entity)
```

Comment:
Update methods Create, Update, Delete to receive array of models

Reply:
done